### ISSUE TITLE
Possible to add new turn alert

### ISSUE THREAD

**Issue by randoomdude** [BullyWiiPlaza/JMaster-Duel-Bot#72]
Possible to many add an alert if you are alt tabed for when a turn changes from opponent to you? Either visual or audio even if client is minimized?

**Comment by BullyWiiPlaza** [comment]
Yes, it's possible. Instead of what you suggested, I think it would be better to alert if the game is not focused and the game expects your input during a duel (this can be activating some card effect or when the opponent passed their turn to you). In those cases your timer starts running so you don't want to miss it for too long.
